common/lib/xmodule/xmodule/templates/problem/latex_problem.yaml

```diff
@@ -2,18 +2,25 @@
 metadata:
```

Comment:
So I'm not sure if anyone ever uses this template. It has a collection of the sample problems from most of the other templates.